libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -142,6 +142,11 @@ func (m *Manager) Apply(pid int) (err error) {
 		// created then join consists of writing the process pids to cgroup.procs
 		p, err := d.path(sys.Name())
 		if err != nil {
+			// The non-presence of the devices subsystem is
+			// considered fatal for security reasons.
+			if sys.Name() == "devices" {
+				return err
+			}
 			if cgroups.IsNotFound(err) {
```

Comment:
I just prefer to link exceptions in the same if statement so it's all in one place. It's a bit of a trivial case here, but still. ;)